frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -26,15 +26,13 @@ import Mode from "metabase-lib/lib/Mode";
 import { isStandard } from "metabase-lib/lib/queries/utils/filter";
 import { isFK } from "metabase-lib/lib/types/utils/isa";
 import { memoizeClass, sortObject } from "metabase-lib/lib/utils";
-/* eslint-enable import/order */
```

Comment:
![Screen Shot 2022-10-13 at 11 25 42 AM](https://user-images.githubusercontent.com/13057258/195641458-17c91266-266c-4d27-ab71-51f4e6eb9579.png)

Any clue what's going wrong here? Tweaking the deps outside of this eslint-disable / eslint-enable import/order block causes eslint to complain about import order.